packages/editor/src/client/blocks/eventBlock.tsx

```diff
@@ -0,0 +1,83 @@
+import PencilIcon from '@rsuite/icons/legacy/Pencil'
```

Comment:
Copied from CommentBlock and adapted for EventBlock

Reply:
Could we re-use the code with a more generic and re-usable component, instead of copying code? I always panic, when I read "copy code" =)